src/auth/auth.ts

```diff
@@ -460,31 +463,31 @@ authStore.subscribe((state: AuthState) => {
 authStore.subscribe(
   withErrorReporting('Error checking groups for timeout status')(async (state: AuthState, oldState: AuthState) => {
     if (userCanNowUseTerra(oldState, state)) {
-      const isTimeoutEnabled = _.some({ groupName: 'session_timeout' }, await Ajax().Groups.list());
+      const isTimeoutEnabled = _.some({ groupName: 'session_timeout' }, Groups().list());
```

Comment:
Lost the `await` here, which looks necessary.

```suggestion
      const isTimeoutEnabled = _.some({ groupName: 'session_timeout' }, await Groups().list());
```

Reply:
🦅 👀